package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "dev": "yarn workspace @okta/vuepress-site dev",
     "build": "yarn stylelint && yarn workspace @okta/vuepress-site build",
+    "preview": "yarn workspace @okta/vuepress-site build",
```

Comment:
maybe `build` can use the `preview` script?

Reply:
Wdym?

Tbh this doesn't save much time, I'd be curious to see the difference once eslint is in there too.

Reply:
i was thinking
```
    "build": "yarn stylelint && yarn preview",
```